composer.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "netzstrategen/image-disclaimer",
```

Only the wordpress prefix is removed.
```suggestion
    "name": "netzstrategen/woocommerce-image-disclaimer",
```